inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/LayerStatistics.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.search;
+
+public class LayerStatistics
+{
+    /** The stats to be calculated */
+    public static final String STATS = "n,sum,min,max,mean,median,standarddeviation";
+
+    private long total;
+    private long maximum;
+    private long minimum;
+    private double mean;
+    private double median;
+    private double standardDeviation;
+
+    private double maximumPerSentence;
+    private double minimumPerSentence;
+    private double meanPerSentence;
+    private double medianPerSentence;
+    private double standardDeviationPerSentence;
+
+    private long noOfDocuments;
+
+    public LayerStatistics()
```

Comment:
Why the empty constructor and no setter?

Reply:
This is just a kind of workaround. I need to access STATS in mtasDocumentIndex to compute the values to put in the actual constructor. Therefore I added this empty constructor. Maybe there is a smarter way to do this.

Reply:
STATS is static final, you why cant you write `LayerStatistics.STATS`?

Reply:
You are right, missed this somehow...